src/cfront.c

```diff
@@ -1693,6 +1763,7 @@ void read_lvalue(lvalue_t *lvalue,
     }
 }
 
+/* add_inst: change pended */
```

Comment:
I didn't get it. Improve the comment.

Reply:
Some changes are still continuing, check [5106846](https://github.com/sysprog21/shecc/pull/85/commits/5106846760c656f87fbfdec955b0467826e65293). I'll complete them soon.